src/main/java/org/mongeez/MongeezRunner.java

```diff
@@ -50,23 +50,23 @@ public void execute() {
         Mongeez mongeez = new Mongeez();
         mongeez.setMongo(mongo);
         mongeez.setDbName(dbName);
-        
+
         if(changeSetsValidator != null) {
             mongeez.setChangeSetsValidator(changeSetsValidator);
         }
         else {
             mongeez.setChangeSetsValidator(new DefaultChangeSetsValidator());
         }
-        
+
         if (changeSetFileProvider != null) {
             mongeez.setChangeSetFileProvider(changeSetFileProvider);
         } else {
             mongeez.setFile(file);
+        }
 
-            if(!StringUtils.isEmpty(userName) && !StringUtils.isEmpty(passWord)){
-            	MongoAuth auth = new MongoAuth(userName, passWord, authDb);
-                mongeez.setAuth(auth);
-            }
+        if(!StringUtils.isEmpty(userName) && !StringUtils.isEmpty(passWord)){
```

Comment:
Is there an `isNotEmpty` util or equivalent?

Reply:
There is [StringUtils#isNotEmpty](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNotEmpty-java.lang.CharSequence-)

Reply:
Fixed.

Reply:
Done. 👍